src/content/en/updates/2018/11/webcomponents-v0-deprecations.md

```diff
@@ -0,0 +1,102 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the deprecations and removals in Chrome 71 to help you plan.
+
+{# wf_updated_on: 2018-11-26 #}
+{# wf_published_on: 2018-11-13 #}
+{# wf_tags: deprecations,removals,chrome73,webcomponents,origintrials #}
+{# wf_blink_components: Blink>DOM #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: Web Components v0 deprecation and removal in Chrome 73 to help you plan.#}
+
+{% include "web/updates/_shared/see-all-dep-rem.html" %}
```

Comment:
I think* you can safely remove this line as this article is specifically about WebComponents V0.

Reply:
Thanks.
Done.